### PR TITLE
Scottx611x/prettier tool auth page

### PR DESCRIPTION
**Pull request by scottx611x**
Before:
pasted image at 2018_04_25 11_51 pm

After:
screen shot 2018-04-27 at 10 22 44 am

### CODE REVIEW

File: refinery/templates/tool_manager/vis-tool-user-not-allowed.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block title %}
+Visualization Error
```
Comment (jkmarx) [inline refinery/templates/tool_manager/vis-tool-user-not-allowed.html]: the formatting seems off. HTML -> 1 tab/ 2 space. Some blocks' interiors need indenting.

File: refinery/tool_manager/views.py [hunk 2]
```diff
@@ -214,8 +220,11 @@ def _proxy_view(self, request, container_name, url):
...
         )
         if not user_has_access_to_tool(request.user, visualization_tool):
-            return HttpResponseForbidden(
-                "User does not have permission to view Tool: {}".format(
+            return render_vis_tool_user_not_allowed_template(
+                request,
+                visualization_tool.name,
+                "User: {} does not have permission to view Tool: {}".format(
+                    request.user.username,
                     visualization_tool.uuid
```
Comment (jkmarx) [inline refinery/tool_manager/views.py]: Would the tool name be more useful than the tool_uuid?

Comment (scottx611x) [reply refinery/tool_manager/views.py]: We still have the Tool name listed in the sub header. I figured it was useful to include something unique about the Tool that erred to help someone debug if they got a screen shot from an end user

Comment (jkmarx) [reply refinery/tool_manager/views.py]: That's a fair point. I'd recommend including the tool name in the main body also. We are getting rid of the subheader soon (1-2 milestones).

**Pull request merged**